app.js

```diff
@@ -47,6 +50,7 @@ app.use((err, request, response, next) => {
 });
 
 app.use((err, request, response, next) => {
+  console.log(err);
```

Comment:
Please remove console.logs, unless it's the one in the 500 server error handler :)